app/services/arcgis_api/geocoder.rb

```diff
@@ -23,6 +30,25 @@ def suggest(text)
       )
     end
 
+    # Makes HTTP request to find an exact address using magic_key
+    # Requires a magic_key returned from a previous response from #suggest
+    # @param magic_key [String]
+    # @return [Array<AddressCandidate>] AddressCandidates
+    def find_address_candidates(magic_key)
+      url = "#{root_url}/findAddressCandidates"
+      params = {
+        magicKey: magic_key,
+        outFields: '*',
```

Comment:
Sorry, I think I forgot to delete this comment, since on closer examination of the documentation, I realized that we'd have to list out quite a few of these `outFields`.